command/resource/apply/apply.go

```diff
@@ -74,17 +77,23 @@ func (c *cmd) Run(args []string) int {
 		}
 	}
 
+	input := c.filePath
```

Comment:
OK, so both the first positional argument _and_ `-f <name>` work? Or did you want to get rid of the `-f` flag altogether?

Reply:
Yeah, both of them work now!